contrib/test/venv-ansible-playbook.sh

```diff
@@ -46,11 +46,14 @@ else
     trap 'rm -rf "$PIPCACHE"' EXIT
 fi
 
+# Create a directory to contain logs and test artifacts
+mkdir -p "$WORKSPACE/results"
+
```

Comment:
This may need coordination with #927 - I'm not married to the location, but we should pick something and stick to it.